### PR TITLE
Section on workspaces in Overview

### PR DESCRIPTION
**Pull request by marc-tonsen**

### CODE REVIEW

File: src/cloud/README.md [hunk 1]
```diff
@@ -57,6 +57,63 @@ Use advanced search in the recordings view to explore and filter your recordings
...
+Public sharing of data with people outside of a workspace is currently not possible.
+
+### Frequently Asked Questions
+
+**When should I use workspaces?**
+
+Workspaces can have many uses. Typical ones are:
+Projects or Studies: Use a separate workspace for every project in order to keep data cleanly separated.
+Teams: If you are a member of multiple teams or organizations, you can use multiple workspaces to control what data you are sharing with whom.
+Clients: If you are working with clients, you can use multiple workspaces in order to keep data separated and to control which collaborator should have access to which client’s data.
```
Comment (papr) [inline src/cloud/README.md:81]: I feel like this should be an explicit enumeration to make the use cases visually more distinct:
```suggestion
- Projects or Studies: Use a separate workspace for every project in order to keep data cleanly separated.
- Teams: If you are a member of multiple teams or organizations, you can use multiple workspaces to control what data you are sharing with whom.
- Clients: If you are working with clients, you can use multiple workspaces in order to keep data separated and to control which collaborator should have access to which client’s data.
```
Optionally, with bold headers
```suggestion
- **Projects or Studies**: Use a separate workspace for every project in order to keep data cleanly separated.
- **Teams**: If you are a member of multiple teams or organizations, you can use multiple workspaces to control what data you are sharing with whom.
- **Clients**: If you are working with clients, you can use multiple workspaces in order to keep data separated and to control which collaborator should have access to which client’s data.
```

File: src/cloud/README.md [hunk 2]
```diff
@@ -57,6 +57,63 @@ Use advanced search in the recordings view to explore and filter your recordings
...
 recording:(test OR stairs) label:bkk wearer:will template:"named recording" duration:<=10m recorded:>now-1y
 ```
+
+## Workspaces
+Workspaces are containers for your data and enable you to share data in Pupil Cloud with your collaborators. All data in Pupil Cloud is part of a workspace and the workspace owner can control access to it. This includes recordings, wearers, templates, labels, projects, and enrichments, which are all isolated within a workspace. 
+
+### Access Control
+Only accounts that have been invited to become members of a workspace are granted access to the data in a workspace. For a more fine-grained control of what each member is allowed to do, they can be assign different roles:
+
+- **Viewer**: This role can view all data, but they are not allowed to edit anything. For example they can download data, playback recordings and view heatmaps, but they can not delete any data, create projects or start compute jobs.
```
Comment (willpatera) [inline src/cloud/README.md:67]:
```suggestion
- **Viewer**: This role can view all data, but they are not allowed to edit anything. For example: a viewer can download data, playback recordings and view heatmaps. A viewer can not delete any data, create projects, or start compute jobs.
```

File: src/cloud/README.md [hunk 3]
```diff
@@ -57,6 +57,63 @@ Use advanced search in the recordings view to explore and filter your recordings
...
+- **Editor**: This role has full read and edit access to all data in the workspace, which includes the ability to create enrichments, start compute jobs and delete data.
+- **Admin**: On top of having full read and edit access like Editors, this role can additionally invite and remove workspace members and change their role.
```
Comment (willpatera) [inline src/cloud/README.md:69]:
```suggestion
- **Admin**: Has all permissions as Editor role plus, Admin can invite and remove workspace members and change workspace members role.
```

File: src/cloud/README.md [hunk 4]
```diff
@@ -57,6 +57,63 @@ Use advanced search in the recordings view to explore and filter your recordings
...
+- **Owner**: The owner of a workspace is the one who initially created the workspace. They have Admin rights and their role can not be changed by anyone.
```
Comment (willpatera) [inline src/cloud/README.md:70]:
```suggestion
- **Owner**: The owner of a workspace is the one who initially created the workspace. Owners have all permissions of an Admin plus Owners not be changed or removed by any other collaborator.
```

File: src/cloud/README.md [hunk 5]
```diff
@@ -57,6 +57,63 @@ Use advanced search in the recordings view to explore and filter your recordings
...
+
+Public sharing of data with people outside of a workspace is currently not possible.
```
Comment (willpatera) [inline src/cloud/README.md:72]:
```suggestion
Public sharing of data with people outside of a workspace is **not** currently possible.
```

File: src/cloud/README.md (hunk 1 above)
Comment (willpatera) [reply src/cloud/README.md:81]: go with bold headers @marc-tonsen

**Review by willpatera — CHANGES_REQUESTED** [review]
@marc-tonsen please see my changes and comments.

**Comment by willpatera** [comment]
@marc-tonsen I think this can be merged when we deploy workspaces.

**Pull request merged**